src/gitguardian-interface/gitguardian-status-bar.ts

```diff
@@ -67,6 +68,11 @@ function getStatusBarConfig(status: StatusBarStatus): StatusBarConfig {
         color: "statusBarItem.errorBackground",
         command: "gitguardian.showOutput",
       };
+    case StatusBarStatus.ignoredFile:
```

Comment:
Maybe we should also have a command triggered on statusBar click for this event

Reply:
I'm not sure what we want the command to be since we are in a state similar to `ready`, maybe `openProblems ?

Reply:
No you are right, let's mimick what we do in the "ready" state. 
I was thinking maybe linking the documentation but I'm not sure this behavior is mentioned in the doc.